osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -544,14 +544,14 @@ protected override void Dispose(bool isDisposing)
 
         private int restartCount;
 
-        private const double volume_requirement = 0.01;
+        private const double volume_requirement = 1e-3; // -60 dB
```

Comment:
why is this using dB if _nothing in the volume subsystem is currently dB-based_, to the best of my knowledge?

Reply:
In both the cases you mentioned, these are arbitrary volume thresholds. When I'm listening on my headphones, -60 decibels seems reasonably quiet, and -30 decibels seems reasonably audible, but they can be any value you want them to be. You can translate any amplitude to decibels, that's just the unit of measurement I'm using to describe it.

-60 dB isn't a level that can be reached with just one volume slider, but a combo of 1% and 10% would reach that level.

Reply:
> -60 dB isn't a level that can be reached with just one volume slider, but a combo of 1% and 10% would reach that level.

honestly at that point i think the notification may as well be removed because i'm not sure anyone is ever going to see it anymore.

Reply:
I can bring it back up to 0.01

Reply:
please do.